pkg/util/provider/machinecontroller/machine_util_test.go

```diff
@@ -2363,7 +2363,7 @@ var _ = Describe("machine_util", func() {
 				},
 				expect: expect{
 					retryPeriod:   machineutils.ShortRetry,
-					err:           errors.New("machine creation is successful. Machine State has been UPDATED"),
+					err:           errors.New("machine creation is successful. Machine Phase/Conditions have been UPDATED"),
```

Comment:
```suggestion
					err:           errSuccessfulPhaseUpdate
```

Similarly for other places as well